flow360/component/flow360_params/boundaries.py

```diff
@@ -234,6 +234,7 @@ class MassInflow(BoundaryWithTurbulenceQuantities):
 
     type: Literal["MassInflow"] = pd.Field("MassInflow", const=True)
     mass_flow_rate: PositiveFloat = pd.Field(alias="massFlowRate")
+    total_temperature_ratio: PositiveFloat = pd.Field(alias="totalTemperatureRatio")
```

Comment:
is there a range for this? Or is positive enough?

Reply:
Just positive is enough. In the formula where totalTemperatureRatio gets used, I don't see any other restriction which limits its range.